editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/utils/MavenUtils.java

```diff
@@ -123,14 +123,15 @@ protected void updateMavenDependencies(
 		final Model model = readMavenModel(pomFile);
 		List<Dependency> deps = getDependencies(project, model);
 
-		// then check if component dependency is already a dep
-		ArrayList<org.fusesource.ide.camel.model.service.core.catalog.Dependency> missingDeps = new ArrayList<>();
-		String scope = null;
-		for (org.fusesource.ide.camel.model.service.core.catalog.Dependency conDep : compDeps) {
-			boolean found = false;
-			for (Dependency pomDep : deps) {
-				if (scope == null && pomDep.getGroupId().equalsIgnoreCase(CAMEL_GROUP_ID)
-						&& pomDep.getArtifactId().equalsIgnoreCase(CAMEL_CORE_ARTIFACT_ID)) {
+        // then check if component dependency is already a dep
```

Comment:
effectively, seems that indentation was broken while resolving conflicts, I will have a closer look on my next conflict that I will resolve